src/backend/launcher.ts

```diff
@@ -526,28 +544,84 @@ async function runWineCommand(
   }
 
   const wineBin = wineVersion.bin.replaceAll("'", '')
-  let finalCommand = `"${wineBin}" ${command}`
+  let finalCommand = `${command}`
   if (additional_command) {
     finalCommand += ` && ${additional_command}`
   }
 
   logDebug(['Running Wine command:', finalCommand], {
-    prefix: LogPrefix.Legendary
+    prefix: LogPrefix.Backend
   })
-  return execAsync(finalCommand, { env: env_vars })
-    .then((response) => {
-      logDebug(['Ran Wine command:', finalCommand], {
-        prefix: LogPrefix.Legendary
-      })
-      return response
+
+  return new Promise((res) => {
```

Comment:
Not that big of a fan of all the repeated code here, *someone* should write a general "runCommand" wrapper soon:tm:

Reply:
I agree. Let's work on that in some refactor in the future.
Maybe a PR to address only this 👍